modules/Hlsjs/resources/hlsjs.js

```diff
@@ -291,11 +294,13 @@
 			backToLive: function () {
 				var _this = this;
 				var vid = this.getPlayer().getPlayerElement();
-				vid.currentTime = vid.duration;
+				this.embedPlayer.goingBackToLive = true;
+				vid.currentTime = vid.duration - this.fragmentDuration * 3;
```

Comment:
@itaykinnrot - maybe add default value 0 or 10 to this, in case frag change doesn't arrive yet, or doesn't update the value.